internal/upgradecheck/header_test.go

```diff
@@ -34,7 +34,10 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/uuid"
 	"k8s.io/apimachinery/pkg/version"
+
+	// Google Kubernetes Engine / Google Cloud Platform authentication provider
 	"k8s.io/client-go/discovery"
+	_ "k8s.io/client-go/plugin/pkg/client/auth/gcp"
```

Comment:
⛏️ Something moved the comment away from the import. I remember something irritating about this... a linter does the wrong thing, maybe?

Reply:
Yeah I had them together at the top and the linter moved the import without the comment. If I put the comment under the discovery import they seem to stick together